src/Mod/Part/App/BRepFeat/MakePrismPyImp.cpp

```diff
@@ -53,7 +53,7 @@ int MakePrismPy::PyInit(PyObject* args, PyObject* kwds)
     PyObject* Direction;
     int Fuse;
     PyObject* Modify;
-    static char* keywords[] = {"Sbase", "Pbase", "Skface", "Direction", "Fuse", "Modifiy", nullptr};
+    static char* keywords[] = {"Sbase", "Pbase", "Skface", "Direction", "Fuse", "Modify", nullptr};
```

Comment:
code modification  (there is an exact similar fix on Line 113 below)
CC @wwmayer

Reply:
The change is OK. Thanks.